### ISSUE TITLE
`publishConfig.directory` package.json configuration breaks publishing

### ISSUE THREAD

**Issue by tunnckoCore** [pikapkg/pack#61]
I'm not sure if this is npm official, or it's a lerna thing, but we can define `publishConfig.directory` to be used as `--contents`.

Before few days when I was playing with pika, I was using that field because pika creates `pkg/` dir, so when I want to publish manually (without cd-ing to pkg dir) I need that field. But because pika copied that field too, my packages was broken, because in the `pkg/` dir there's no another `pkg` dir. 

So, basically, just when handling the manifest thing, delete the `directory` field from `publishConfig`.
